appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/annotations/FloatRange.kt

```diff
@@ -0,0 +1,41 @@
+package com.bumble.appyx.interactions.core.annotations
+
+/**
```

Comment:
This I assume was copied from https://androidx.de/androidx/annotation/FloatRange.html or similar.

It has an Apache 2.0 license. Let's mention and link to the original.